payments/stripe/forms.py

```diff
@@ -20,29 +22,53 @@ def __init__(self, *args, **kwargs):
             self.payment.change_status('rejected')
             raise RedirectNeeded(self.payment.get_failure_url())
 
+    def _handle_potentially_fraudulent_charge(self, charge, commit=True):
+        fraud_details = charge['fraud_details']
+        if fraud_details.get('stripe_report', None) == 'fraudulent':
+            reject_fraud_choice = FRAUD_CHOICES[2][0]
```

Comment:
It's intention was to make it less probable to be skipped in case of a potential, future substitution with an enum. I've seen a previous change that renamed a `confirmed` status to `accept`, so I picked this approach to mitigate that risk (but main for the dependency on the `FRAUD_CHOICES` import).